sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -110,22 +111,22 @@ def create_properties(user_agent=None):
     :rtype: dict
     """
     properties = {}
-    properties[types.AMQPSymbol("product")] = USER_AGENT_PREFIX
-    properties[types.AMQPSymbol("version")] = VERSION
```

Comment:
I don't think we can just drop the AMQPSymbol from these.... otherwise I think this is just going to be encoded as strings. I see this in a couple of places, so we should double check the payloads are being encoded as we think they are.